### PR TITLE
HotFix for Python 3.11 

### PR DESCRIPTION
**Pull request by MateusStano**
## Pull request type

  - [x] Lint (`black rocketpy`) has passed locally and any fixes were made
  - [x] All tests (`pytest --runslow`) have passed locally

## What is the current behavior?

RocketPy can not be installed with Python 3.11. This is due to the netCDF4 lib version requirement being limited to `>=1.4, 1.6<=`

## What is the new behavior?

The netCDF4 library has been updated to `>=1.6.2`

## Does this introduce a breaking change?

- [ ] Yes
- [x] No

### CODE REVIEW

**Comment by giovaniceotto** [comment]
I'll check if this works in Google Colab before approving, due to this: https://github.com/Unidata/netcdf4-python/issues/1179

**Comment by giovaniceotto** [comment]
Still not working in Windows. At least, the Windows test is not passing, due to an hdf5 error raised when trying to install netCDF4.

**Comment by giovaniceotto** [comment]
Here is what I have done:

 - [x] Fixed support for Python 3.7
 - [x] Bumped up version to 0.13.1 

Here what still needs to be done:

- [ ] Find a solution that works in Google Colab

**Comment by giovaniceotto** [comment]
Good news:

 - Found a solution that works for Google Colab in https://github.com/RocketPy-Team/RocketPy/pull/348/commits/3c6f115c5ccc988e4497a78cdcc407c3a36b507b
 
Bad news:

- Google Colab's problem seems to also happen in Codespaces.

I wonder if the issue with netCDF4 version (https://github.com/Unidata/netcdf4-python/issues/1179) happens in any linux installation. @phmbressan, can you help us out here?

File: docs/user/requirements.rst [hunk 1]
```diff
@@ -21,18 +20,19 @@ The following packages are needed in order to run RocketPy:
 - Numpy >= 1.0
 - Scipy >= 1.0
 - Matplotlib >= 3.0
-- netCDF4 >= 1.4
+- netCDF4 >= 1.4, < 1.6 for Python 3.7+, netCDF4 >= 1.6.2 for Python 3.11
```
Comment (Gui-FernandesBR) [inline docs/user/requirements.rst:23]: is this code line already working properly?

Comment (giovaniceotto) [reply docs/user/requirements.rst:23]: This is documentation. It shouldnbe working. Are you getting errors when building the docs?

**Review by giovaniceotto — APPROVED** [review]

**Pull request merged**